tests/conftest.py

```diff
@@ -2890,6 +2890,47 @@ def finalizer():
     return factory
 
 
+@pytest.fixture()
+def multi_snapshot_factory(snapshot_factory):
+    """
+    Snapshot factory. Calling this fixture creates sets of volume snapshot from the
+    specified PVC
```

Comment:
This gives an impression that multiple snapshots will be created from the same PVC.